README.md

```diff
@@ -97,53 +97,75 @@ You can also use it within the browser; install via npm and use the `check-error
 
 The primary export of `check-error` is an object which has the following methods:
 
-* `compatibleInstance(err, errorLike)` - Checks if an error is compatible with another `errorLike` object. If `errorLike` is an error instance we do a strict comparison, otherwise we return `false` by default, because instances of objects can only be compatible if they're both error instances.
-* `compatibleConstructor(err, errorLike)` - Checks if an error's constructor is compatible with another `errorLike` object. If `err` has the same constructor as `errorLike` or if `err` is an instance of `errorLike`.
-* `compatibleMessage(err, errMatcher)` - Checks if an error message is compatible with an `errMatcher` RegExp or String (we check if the message contains the String).
+* `checkError(errObj[, errLike[, errMsgMatcher]])` or `checkError(errObj[, errMsgMatcher])` - Checks if an `Error` instance matches criteria defined by `errLike` and/or `errMsgMatcher`.
+    * If `errLike` is an `Error` constructor, then validate that `errObj` is an instance of `errLike`.
+    * If `errLike` is an `Error` instance, then validate that `errObj` is strictly (`===`) equal to `errLike`.
```

Comment:
💭  Given some of the talk in the chai issue tracker, and around deep-eql, should we losen this restriction to say

> If `errLike` is an `Error` instance, then validate that `errObj` has the same `constructor`, `message` and any other enumerable properties (excluding `stack`)?

Reply:
I'm in favor of this change. But I think it'd need to be implemented in `deep-eql` and imported here. If one of the enumerable properties is an object, then I believe the normal deep equality algorithm should apply to that property.

Reply:
Yes, well said, let's leave it out of scope here, and work on putting this in `deep-eql` and bubbling up through the codebase.